tests/framework/clients/rancher/v1/client.go

```diff
@@ -283,6 +287,45 @@ func (c *NamespacedSteveClient) Update(existing *SteveAPIObject, updates any) (*
 	return c.SteveClient.Update(existing, updates)
 }
 
+func (c *NamespacedSteveClient) PerformPutCaptureHeaders(host, token, endpoint, namespace, name string, payload interface{}) (http.Header, []byte, error) {
```

Comment:
Andrew I believe if it's the namespace client you don't need the namespace as a parameter. Can you look at the other member functions to see.

Reply:
Ah, good catch! Thank you! 😁

Fixed this so namespace isn't a parameter 👍